### PR TITLE
update python-docs

### PR DESCRIPTION
**Pull request by magnurud**

### CODE REVIEW

File: reference/python-sdk/v3.2.md [hunk 1]
```diff
@@ -0,0 +1,1472 @@
...
+>>> client = Client()
+>>> in_schema = {'$schema': 'https://json-schema.org/draft-04/schema#', 'title': 'in', 'properties': {...} }
+>>> out_schema = {'$schema': 'https://json-schema.org/draft-04/schema#', 'title': 'out', 'properties': {...} }
+>>> # A typical docker transitions
+>>> docker_params = {
+>>>     'imageUrl': '<image_url>',
+>>>     'credentials': {'username': '<username>', 'password': '<password>'}
+>>> }
+>>> client.create_transition('docker', in_schema=in_schema, out_schema=out_schema, params=docker_params)
+>>> # A typical manual transitions
```
Comment (augustak) [inline reference/python-sdk/v3.2.md]: What about 'A manual transition with UI' to give som context to the jsRemoteComponent part?

File: reference/python-sdk/v3.2.md [hunk 2]
```diff
@@ -0,0 +1,1472 @@
...
+Creates a transition handle, calls the POST /transitions endpoint.
+
+```python
+>>> import json
+>>> from pathlib import Path
+>>> from las.client import Client
+>>> client = Client()
+>>> in_schema = {'$schema': 'https://json-schema.org/draft-04/schema#', 'title': 'in', 'properties': {...} }
+>>> out_schema = {'$schema': 'https://json-schema.org/draft-04/schema#', 'title': 'out', 'properties': {...} }
+>>> # A typical docker transitions
```
Comment (augustak) [inline reference/python-sdk/v3.2.md]: transitionS -> transition

File: getting-started/dev/python.md [hunk 3]
```diff
@@ -16,30 +16,36 @@ Suppose we wish to run inference on a document using Lucidtech’s invoice model
 from las import Client
 
 client = Client()
-document = client.create_document('invoice.pdf')
-prediction = client.create_prediction(document_id=document.id, model_name='invoice')
+document = client.create_document('invoice.pdf', 'application/pdf')
+prediction = client.create_prediction(document_id=document['documentId'], model_id='las:model:<hex>')
```
Comment (augustak) [inline getting-started/dev/python.md]: Maybe be consistent with keywords on these two lines?

File: reference/python-sdk/v3.2.md [hunk 4]
```diff
@@ -0,0 +1,1472 @@
...
+>>> specification = {'language': 'ASL', 'version': '1.0.0', 'definition': {...}}
+>>> error_config = {'email': '<error-recipient>'}
+>>> client.create_workflow(specification, error_config=error_config)
+```
+
+
+* **Parameters**
+
+    
+    * **specification** (*dict*) – Specification of the workflow,
```
Comment (augustak) [inline reference/python-sdk/v3.2.md]: Is it an idea to quickly explain with features from ASL is supported / not supported and what the 'resource' key should be in our case?  (i.e. a transition id)

File: reference/python-sdk/v3.2.md (hunk 1 above)
Comment (magnurud) [reply reference/python-sdk/v3.2.md]: This is autogenerated from the python-sdk, so comments on this file should be done there :P

File: reference/python-sdk/v3.2.md (hunk 4 above)
Comment (magnurud) [reply reference/python-sdk/v3.2.md]: I think that belongs in a tutorial and not in a method specification don't you agree?

Comment (augustak) [reply reference/python-sdk/v3.2.md]: Yes agreed

**Review by augustak — APPROVED** [review]

**Pull request merged**